pkg/operator/operator.go

```diff
@@ -172,22 +167,32 @@ func (o *Operator) watchTPR(watchVersion string) error {
 
 }
 
+func startCluster(cluster *Cluster) {
+	err := cluster.CreateInstance()
+	if err != nil {
+		logger.Errorf("failed to create cluster in namespace %s. %+v", cluster.Namespace, err)
+	}
+}
+
 func (o *Operator) findAllClusters() (string, error) {
 	logger.Info("finding existing clusters...")
 	clusterList, err := getClusterList(o.clientset.CoreV1().RESTClient(), o.Namespace)
 	if err != nil {
 		return "", err
 	}
-
+	logger.Infof("found %d clusters", len(clusterList.Items))
 	for i := range clusterList.Items {
 		c := clusterList.Items[i]
 
 		stopCh := make(chan struct{})
 		ns := c.Spec.Namespace
-		newCluster := newCluster(ns, c.Spec.Version, c.Spec.UseAllDevices, o.factory, o.clientset)
+		existingCluster := newCluster(ns, c.Spec.Version, c.Spec.UseAllDevices, o.factory, o.clientset)
 		o.stopChMap[ns] = stopCh
-		o.clusters[ns] = newCluster
+		o.clusters[ns] = existingCluster
 		o.clusterRVs[ns] = c.Metadata.ResourceVersion
+
+		logger.Infof("resuming cluster %s in namespace %s", c.Metadata.Name, ns)
+		go startCluster(existingCluster)
```

Comment:
what are the semantics of calling startCluster on an existing cluster? it doesn't look like we were doing that before, will everything be idempotent so the existing cluster resumes or continues running OK?